docs/Testing.md

```diff
@@ -76,15 +80,18 @@ func @main_graph(%arg0: tensor<3x4x5xf32>, %arg1: tensor<3x4x5xf32>) -> tensor<3
 ```
 with `IMPORTER_FORCE_DYNAMIC='-1:-1'`, the result is:
 ```
-func @main_graph(%arg0: tensor<?x?x?xf32>, %arg1: tensor<?x?x?xf32>) -> tensor<?x?x?xf32>
+func @main_graph(%arg0: tensor<?x?x?xf32>{onnx.name = "x“, onnx.dim_params = "0:a,1:b,2:c"}, %arg1: tensor<?x?x?xf32>{onnx.name = "y“, onnx.dim_params = "0:a,1:b,2:c"}) -> tensor<?x?x?xf32>
```

Comment:
`-1` for `dimIndex` to say all arguments have the same dynamic dims does not make sense. It is too restricted.
Let's consider the dynamic tests for CPU where it supports broadcasting, e.g `add(tensor<3x5xf32>, tensor<3x1xf32>`, using `-1:-1` will force all dynamic dimensions being the same and the lowering will generate different code, not taking care of broadcasting.

In summary, I recommend to not support this, and let users specify the same dynamic dimensions manually for the cases they need.

Reply:
I removed the assumption about default relationships among input data ranks, and specified the relationship manually in the test/accelerators/NNPA/backend/CMakeLists.txt file for each dynamic backend tests on NNPA.
Thanks for the suggestions.